tests/requirements-relaxed.txt

```diff
@@ -14,15 +14,17 @@ aiosignal==1.3.1
     # via aiohttp
 alabaster==0.7.16
     # via sphinx
+annotated-types==0.6.0
+    # via pydantic
 ansible-pygments==0.1.1
     # via
     #   antsibull-docs
     #   sphinx-ansible-theme
-antsibull-core==2.2.0
+antsibull-core==3.0.0
```

Comment:
This is a major version bump, but I don't expect it to be problematic[^1]

[^1]: Famous last words...

Reply:
Ha. Yeah this one could get spicy but we're not backporting and can downgrade again if needed. It's good to shine a light on this bump so we can be on the lookout for breakage.